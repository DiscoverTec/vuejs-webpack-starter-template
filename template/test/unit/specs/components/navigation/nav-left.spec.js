import { shallow } from '@vue/test-utils'
import { createRenderer } from 'vue-server-renderer'
import Component from '@/components/navigation/nav-left.vue'

describe(`Component ${Component.name}`, () => {

  it('renders to a snapshot', () => {
    const renderer = createRenderer()
    const wrapper = shallow(Component, {})
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err)
      expect(str).toMatchSnapshot()
    })
  })
})
