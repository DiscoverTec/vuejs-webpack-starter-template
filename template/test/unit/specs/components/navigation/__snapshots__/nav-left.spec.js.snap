// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component nav-left renders to a snapshot 1`] = `
<nav class="nav flex-column">
    <router-link to="/" title="Main" class="nav-link">
        <i aria-hidden="true" class="fa pr-2 fa-puzzle-piece"></i>
        Main
    </router-link>
</nav>
`;
